tests/investments/test_serializer_create.py

```diff
@@ -33,7 +33,7 @@ def test_valid_subscription_creation(self):
             for field, errors in serializer.errors.items():
                 print(f"Validation errors in field '{field}': {errors}")
         self.assertTrue(is_valid, "Serializer validation failed")
-        subscription = serializer.save()
+        subscription = serializer.save(investor=self.investor1)
```

Comment:
Passing investor via save() but serializer may not expect it if context already supplies it. Ensure serializer signature and test usage are consistent.